src/user-icon/classes.scss

```diff
@@ -25,3 +25,4 @@
 .iui-user-icon-list {
   @include iui-user-icon-list;
 }
+
```

Comment:
?

Reply:
You comment whats wrong, do not ask her, whats wrong...